nl.json

```diff
@@ -72,15 +71,21 @@
                 "discord" : "Discord",
                 "twitter" : "Twitter"
             },
-            "api" : "api",
+           "api" : "api",
```

Comment:
The spacing was correct before. Please don't remove the space before the translation key.